### PR TITLE
Add base styling and favicon

### PR DESCRIPTION
**Pull request by SeaJerry**
Porting the base Bootstrap CSS to TailwindCSS styling on the DSC's current webpage and repo. This includes font-family and color palette. We will still need to figure out how to create a parallax, the styling for the parallax feature is ported over and commented out until we figure out a solution. Resolves issue https://github.com/kindsys/digital-services-coalition/issues/13

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -3,47 +3,294 @@
 @import 'tailwindcss/utilities';
 
 @layer components {
-    /* h2 styles */
+    body {
+        margin: 0;
+        padding: 0;
+        color: gray;
+        font-family: 'Open Sans', Calibri, Source Sans Pro, sans-serif;
+        font-size: 11pt;
```
[the remark this answers is not in the document]
Comment (SeaJerry) [reply src/styles/main.css:11]: Not sure, All of the CSS that is in the `main.css` is what is migrated over from the original DSC repo.

[the remark this answers is not in the document]
Comment (tridnguyen) [reply src/styles/main.css:11]: Agree with @gracexu94 here, the value should be converted to px, em or rem.  Different browsers are going to interpret `pt` differently.  So, you won't have any consistency between browsers.

**Comment by tridnguyen** [comment]
In general, you're going to want to use `em` or `rem` instead of `px` in a lot of the places where you're specifying padding and margins.  That way if the user zooms in/out, the padding and margins will adjust with it.  You'll want to use `px` if you don't want those values to scale with the text.  I generally avoid using `px` in most scenarios if I can.  If your base font size is 16px (most modern browsers use this as the default), then just divide by it and use `rem`.  If you want the padding to be relative to the parent element, use `em`.

File: src/styles/main.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (SeaJerry) [reply src/styles/main.css]: That makes sense, Thanks Grace and Tri!

**Review by gracexu94 — APPROVED** [review]
Thanks for making the changes, LGTM!

**Review by tridnguyen — APPROVED** [review]

**Pull request merged**